05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+$$se(\hat{\theta})=\sqrt{\frac{1}{R} \sum_{r=1}^R \left( \hat{\theta}_r-\bar{\theta}\right)^2}$$ where $$\bar{\theta}=\frac{1}{R}\sum_{r=1}^R \hat{\theta}_r$$
+
+The default option for `mse` is to use the global option of "survey.replicates.mse" which is set to `FALSE` initially unless a user changes it. Unless documentation states otherwise, for BRR, set `mse` to `TRUE`.
+
+#### The syntax {-} 
+
+Replicate weights generally come in groups and are sequentially numbered such as PWGTP1, PWGTP2, ..., PWGTP80 in the American Community Survey (ACS) or BRRWT1, BRRWT2, ..., BRRWT96 in the 2015 Residential Energy Consumption Survey (RECS). The {srvyr} package relies on tidy selection^[dplyr documentation on tidy-select: https://dplyr.tidyverse.org/reference/dplyr_tidy_select.html] to choose variables. If replicate weight variables need to be specified with a character vector, use the `all_of` function to select variables from a character vector. Some other methods are also illustrated below in the examples but these apply to any type of replicate weights and not just BRR.
+
+If a dataset had WT0 for the main weight and had 20 BRR weights indicated WT1, WT2, ..., WT20, use the following syntax:
+
+```{r}
+#| label: repgen
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT0, repweights= all_of(str_c("WT", 1:20)),
+                type="BRR", mse=TRUE)
+
+des_brr <- dat %>%
+  as_survey_rep(weights = WT0, repweights= num_range("WT", 1:20),
+                type="BRR", mse=TRUE)
+
+```
+
+If a dataset had WT for the main weight and had 20 BRR weights indicated REPWT1, WT2, ..., REPWT20, the following syntax could be used (both are valid):
+
+```{r}
+#| label: brr
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= all_of(str_c("REPWT", 1:20)),
+                type="BRR", mse=TRUE)
+
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= starts_with("REPWT"),
+                type="BRR", mse=TRUE)
+```
+
+If the replicate weight variables are on the file consecutively, the following syntax can also be used:
+
+```{r}
+#| label: brr2
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= REPWT1:REPWT20, type="BRR",
+                mse=TRUE)
+```
+
+Typically, the replicate weights sum to a value similar to the main weight as they are both supposed to provide population estimates. Rarely, an alternative method will be used where the replicate weights have values of 0 or 2 in the case of BRR weights. In this case, the replicate weights are not combined and the option `combined_weights = FALSE` should be indicated with the default value being TRUE. This specific syntax is shown below:
```

Comment:
```suggestion
Typically, the replicate weights sum to a value similar to the main weight as they are both supposed to provide population estimates. Rarely, an alternative method will be used where the replicate weights have values of 0 or 2 in the case of BRR weights. In this case, the replicate weights are not combined and the option `combined_weights = FALSE` should be indicated, as the default value for this argument is TRUE. This specific syntax is shown below:
```

Reply:
I may have missed this, but I don't think you talk anywhere about what "combined" means.  Might want to add a sentence about this.